PropertyChanged.Fody/PropertyDataWalker.cs

```diff
@@ -40,6 +40,12 @@ void GetPropertyData(PropertyDefinition propertyDefinition, TypeNode node)
             .Select(x => x.ShouldAlsoNotifyFor);
 
         var backingFieldReference = node.Mappings.First(x => x.PropertyDefinition == propertyDefinition).FieldDefinition;
+
+        if (backingFieldReference?.CustomAttributes.ContainsAttribute("PropertyChanged.DoNotNotifyAttribute") == true)
```

Comment:
doesnt a property always have a FieldDefinition?

Reply:
Apparently not, if we can't find a "canonical" field. This method can return null:

https://github.com/Fody/PropertyChanged/blob/fe1b43330df27abd4d63a0c69ba08581d36a4773/PropertyChanged.Fody/MappingFinder.cs#L72

Reply:
TIL :)